src/Components/WeekdayCardsOnline/WeekdayCardsOnline.module.css

```diff
@@ -0,0 +1,34 @@
+.cards {
+  display: grid;
+  grid-template-columns: 50% 50%;
```

Comment:
Using percantage in grids in combination with paddings, gaps etc. tends to be a bit dangerous as those values can lead to a total of more than 100%. Please consider `1fr 1fr` instead here.